cliapi/cliapi.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python3
+
+# Guide for cliapi developers found here:
+# https://github.com/edlane/cliapi/blob/master/README.md
+
+import os
+if 'PYCHARM_DEBUG_ME' in os.environ:
+    import pydevd
+    # we want debugging enabled up top because much of the cliapi framework
+    # "import-meta-magic" happens before __main__() has ever been called...
+    #
+    # Remote debugging session enabled in cloud environments
+    # by establishing a reverse ssh tunnel with bash:
+    #
+    #   local_host> ssh -R 8282:localhost:8282 lane@remote_host
+    #
+    # Then connect back to your local pycharm debugger with:
+    #
+    #   remote_host> PYCHARM_DEBUG_ME= cliapi [various options]...
+    #
+    pydevd.settrace('localhost', port=8282, stdoutToServer=True,
+                    stderrToServer=True)
+
```

Comment:
There should be a different way of dealing with this. Paying homage to specific tools is not a great option, we generally also do not insert editor directives such as ` # vi: ts=4 expandtab` into our code.